client/my-sites/upgrades/checkout/credit-card-payment-box.jsx

```diff
@@ -44,7 +44,7 @@ var CreditCardPaymentBox = React.createClass( {
 						cart={ this.props.cart }
 						transactionStep={ this.props.transactionStep } />
 
-					{ ! cartItems.hasFreeTrial( cart ) ?
+					{ cartValues.isPayPalExpressEnabled( cart ) ?
```

Comment:
This is perfectly fine, but can be also codes as:
```js
{ cartValues.isPayPalExpressEnabled( cart ) && <a className="credit-card-payment-box__switch-link" href="" onClick={ this.handleToggle }>{ this.translate( 'or use PayPal' ) }</a> }
```

Just sharing what we were using when working on Purchases. This is something to consider in the future :)